pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -257,15 +257,16 @@ def newpkgcb(pkgId, name, arch):
         """
         Build `DeclarativeContent` from the repodata.
         """
-        packages_pb = ProgressBar(message='Parsed Packages')
-        erratum_pb = ProgressBar(message='Parsed Erratum')
+        packages_pb = ProgressBar(message='Parsed Packages', code='parsing.packages')
+        erratum_pb = ProgressBar(message='Parsed Erratum', code='parsing.erratum')
```

Comment:
I know this isn't your fault but we should probably adjust the convention here to be consistent with packages. So I'd recommend replacing erratum_pb and parsing.erratum with errata_pb and parsing.errata.